### ISSUE TITLE
Documentation for importing azurerm_api_management_subscription is wrong.

### ISSUE THREAD

**Issue by rahmnstein** [terraform-providers/terraform-provider-azurerm#8942]
https://www.terraform.io/docs/providers/azurerm/r/api_management_subscription.html

Documentation says:

`terraform import azurerm_api_management_subscription.example /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/example-resources/providers/Microsoft.ApiManagement/service/example-apim/properties/example-apimp`

Should be:
`terraform import azurerm_api_management_subscription.example /subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/example-resources/providers/Microsoft.ApiManagement/service/example-apim/subscriptions/subscription-name`

**Comment by sonic1981** [comment]
I'm not really up to speed on resource ids but:

Looking at a api management I've just created I end up with a resource id of:
```
/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/example-resources/providers/Microsoft.ApiManagement/service/example-apim
```
What are the later sections there for at all?

**Comment by rahmnstein** [comment]
@sonic1981 This is not an "API Management", but an "API Management Subscription".
You can retrieve it with powershell:

`$Context = New-AzApiManagementContext -ResourceId apim-id
Get-AzApiManagementSubscription -Context $Context`

But seems very outdated, I used a null_resource and an external script instead. You cannot assign an API as scope and it forces you to set a user_id which is not required through the portal. The documeentation is still incorrect though.

**Comment by sonic1981** [comment]
Submitted PR https://github.com/terraform-providers/terraform-provider-azurerm/pull/8951

**Comment by tombuildsstuff** [comment]
Fixed via #8951, thanks @sonic1981
